backend/molgenis-emx2-jobs/src/main/java/org/molgenis/emx2/tasks/TaskServiceInMemory.java

```diff
@@ -49,7 +49,7 @@ public void removeOlderThan(long milliseconds) {
     Set<String> keys = tasks.keySet();
     for (String key : keys) {
       if (tasks.get(key).end != 0
-          && tasks.get(key).end < System.currentTimeMillis() - milliseconds) {
+          && tasks.get(key).end <= System.currentTimeMillis() - milliseconds) {
```

Comment:
please add the issue / failing test to the PR description

Reply:
It is the one you reported yourself: failing unit test

```
> Task :backend:molgenis-emx2-jobs:test FAILED



org.molgenis.emx2.tasks.TestTask > test1 FAILED

    internal error: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.molgenis.emx2.tasks.Step.getIndex()" is null (through reference chain: org.molgenis.emx2.tasks.DummyTask["steps"]->java.util.ArrayList[3]->org.molgenis.emx2.tasks.Task["description"])

        at org.molgenis.emx2.tasks.Step.toString(Step.java:134)

        at java.base/java.lang.String.valueOf(String.java:3365)

        at java.base/java.io.PrintStream.print(PrintStream.java:877)

        at org.gradle.internal.io.LinePerThreadBufferingOutputStream.println(LinePerThreadBufferingOutputStream.java:230)

        at org.molgenis.emx2.tasks.TestTask.test1(TestTask.java:21)



        Caused by:

        com.fasterxml.jackson.databind.JsonMappingException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.molgenis.emx2.tasks.Step.getIndex()" is null (through reference chain: org.molgenis.emx2.tasks.DummyTask["steps"]->java.util.ArrayList[3]->org.molgenis.emx2.tasks.Task["description"])

            at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:390)

            at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:349)

            at com.fasterxml.jackson.databind.ser.std.StdSerializer.wrapAndThrow(StdSerializer.java:316)

            at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:778)

            at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178)

            at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serializeContents(IndexedListSerializer.java:119)

            at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serialize(IndexedListSerializer.java:79)

            at com.fasterxml.jackson.databind.ser.impl.IndexedListSerializer.serialize(IndexedListSerializer.java:18)

            at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:728)

            at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:770)

            at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:178)

            at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider._serialize(DefaultSerializerProvider.java:480)

            at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:319)

            at com.fasterxml.jackson.databind.ObjectMapper._writeValueAndClose(ObjectMapper.java:4487)

            at com.fasterxml.jackson.databind.ObjectMapper.writeValueAsString(ObjectMapper.java:3742)

            at org.molgenis.emx2.tasks.Step.toString(Step.java:131)

            ... 4 more



            Caused by:

            java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "org.molgenis.emx2.tasks.Step.getIndex()" is null

                at org.molgenis.emx2.tasks.Step.getDescription(Step.java:57)

                at jdk.internal.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)

                at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

                at java.base/java.lang.reflect.Method.invoke(Method.java:567)

                at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:689)

                at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:770)

                ... 16 more



1 test completed, 1 failed

````